src/service/apiClient.js

```diff
@@ -52,4 +52,4 @@ async function request(method, endpoint, data, auth = true) {
   return response.json();
 }
 
-export { login, getPosts, register, createProfile };
+export { login, getPosts, register, createProfile, request, get, post, patch };
```

Comment:
please don't export `request, get, post, patch` -> those functions are meant to be kept "private" in this module; instead we define new API functions: `getProfile`, `updateProfile` whenever these are needed, so that, in the future, if we want to change say to GraphQL / some other networking library, we don't have to update our whole codebase.